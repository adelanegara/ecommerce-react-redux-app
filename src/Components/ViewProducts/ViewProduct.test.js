import React from "react";
import { screen, render } from "@testing-library/react";
import { ViewProductUnwrapped } from ".";

let wrapper;
beforeEach(() => {
  const { getByTestId } = render(<ViewProductUnwrapped />);
  wrapper = getByTestId("view");
});

test("test navbar", () => {
  expect(wrapper).toBeInTheDocument();
});

test("Test title", () => {
  const wrapper = screen.getByText(/DETAIL PRODUCT/);
  expect(wrapper).toBeInTheDocument();
});

test("rate text", () => {
  const wrapper = screen.getByText(/Rate:/);
  expect(wrapper).toBeInTheDocument();
  expect(wrapper).toMatchSnapshot();
});
