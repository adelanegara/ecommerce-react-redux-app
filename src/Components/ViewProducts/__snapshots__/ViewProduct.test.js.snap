// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rate text 1`] = `
<p>
   Rate: 
</p>
`;
